app/assets/stylesheets/posts.scss

```diff
@@ -217,10 +219,16 @@ h1 .badge.is-tag.is-master-tag {
   font-weight: lighter;
 }
 
+.post--actions {
```

Comment:
Are these classes already in use and we only needed to add this CSS property?  (I expected to find an invocation in the PR.)

Reply:
Yup, nothing new, we just don't have any styling overrides for `post--actions` (well, _now_ we do)